duckduckgo/Android
# Feature/david/fix bottom nav styles

Task/Issue URL: https://app.asana.com/0/1157893581871903/1173860855562443
Task/Issue URL: https://app.asana.com/0/1157893581871903/1173860855561445
Task/Issue URL: https://app.asana.com/0/1157893581871903/1173860855561441

**Description**:
The bottom bar navigation feature introduced a couple of issues on API 21, this PR fixes them.

***Screenshots***:
| Before | After |
| ------ | ----- | 
![bookmark_overflow_color_api21](https://user-images.githubusercontent.com/531613/80961501-5f20f000-8e0b-11ea-9ed0-cca6e2055229.png)|![Screenshot_20200504_132653](https://user-images.githubusercontent.com/531613/80961649-adce8a00-8e0b-11ea-94f0-5a611b59e125.png)|

| Before | After |
| ------ | ----- | 
![tabs_settings_hard_to_tap](https://user-images.githubusercontent.com/531613/80961528-6ba54880-8e0b-11ea-9bfd-47bfd50be349.png)|![Screenshot_20200504_132545](https://user-images.githubusercontent.com/531613/80961591-8ecff800-8e0b-11ea-9adc-3bed2dbcc4ad.png)|

| Before | After |
| ------ | ----- | 
![menu_dark](https://user-images.githubusercontent.com/531613/80961492-5cbe9600-8e0b-11ea-9c55-8f75088a33b3.png)|![Screenshot_20200504_132621](https://user-images.githubusercontent.com/531613/80961619-9ee7d780-8e0b-11ea-8766-4de6e3245ae2.png)|

**Steps to test this PR**:
On API 21:

1. Add a bookmark and check the color of the oveflow
2. Open the popup menu in the Tabs screen, check it's boundaries are correct
3. Open the popup menu in the Browser screen, check rendering of "Desktop site" menu


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
